helm-charts/interoperator/templates/sfcluster.yaml

```diff
@@ -2,17 +2,18 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
-  labels:
-    controller-tools.k8s.io: "1.0"
   name: sfclusters.resource.servicefabrik.io
 spec:
   group: resource.servicefabrik.io
   names:
     kind: SFCluster
+    listKind: SFClusterList
     plural: sfclusters
-  scope: Namespaced
+    singular: sfcluster
+  scope: ""
```

Comment:
`sfcluser` is no more namespaced scoped is it ?

Reply:
"" means Namespaced as it is the default value. So all our resources are still namespace scoped. This change is due to kubebuilder. The file is generated